Madness.xcodeproj/project.pbxproj

```diff
@@ -159,6 +159,7 @@
 				D4FC47B21A37E47C00D23A6F /* Products */,
 			);
 			sourceTree = "<group>";
+			usesTabs = 1;
```

Comment:
It's a project-level setting. If you click on the actual Xcode project in the navigator, then go over to the utilities sidebar on the right, there's an option to set tabs/spaces at the project level, overriding whatever your defaults are in Xcode's preferences.
![screen shot 2015-04-27 at 7 25 30 pm](https://cloud.githubusercontent.com/assets/526054/7359969/47fb842e-ed13-11e4-84a6-48036b01b4eb.png)